kubespawner/spawner.py

```diff
@@ -1642,10 +1642,11 @@ async def progress(self):
                 # pod_id may change if a previous pod is being stopped
                 # before starting a new one
                 # use the uid of the latest event to identify 'current'
-                pod_id = events[-1].involved_object.uid
+                pod_id = events[-1]["involvedObject"]["uid"]
                 for i in range(next_event, len_events):
                     event = events[i]
                     # move the progress bar.
+
```

Comment:
nit: unrelated change (try to avoid these because they can cause merge conflicts and bloat the size of the review)